packages/xstate-wallet/src/constants.ts

```diff
@@ -59,4 +59,4 @@ export const CLEAR_STORAGE_ON_START = getBool(process.env.CLEAR_STORAGE_ON_START
 export const USE_INDEXED_DB = getBool(process.env.USE_INDEXED_DB, false);
 
 export const LOG_FILE = process.env.LOG_FILE;
-export const ADD_LOGS = LOG_FILE ? true : getBool(process.env.ADD_LOGS);
+export const ADD_LOGS = LOG_FILE ? true : getBool(process.env.ADD_LOGS || 'false');
```

Comment:
Ideally, we just use one env var to denote whether we do logging. Now we have two... that would make it confusing to anyone who looks at this.

Reply:
There are two almost orthogonal issues:
1. do you want logs to happen?
2. where should logs go?

An alternative could be a single `LOG_DESTINATION` variable which:
- when `undefined`, turn it off
- when is `'console'`, log to console
- otherwise, assume `LOG_DESTINATION` is a filename, and log to the file

What do you suggest?

Reply:
I like the `LOG_DESTINATION` approach. People don't have to wrap the head around to juggle two vars. Less is better, to me.